commit def410ab86e6e2b4d4df3c090832a249af882188 (refs/changes/63/1263/2)
Author: Richard Bradley <richard.bradley@joyent.com>
Date:   2017-01-17T17:16:56+00:00 (2 years, 9 months ago)
    
    MANATEE-320 update manta-manatee, sdc-manatee for recent fixes
    Reviewed by: Dave Pacheco <dap@joyent.com>
    Approved by: Dave Pacheco <dap@joyent.com>

diff --git a/package.json b/package.json
index ffd3271..d6c8218 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     },
     "dependencies": {
         "manta": "1.2.6",
-        "manatee": "git+ssh://git@github.com:joyent/manatee.git#3771c6fbd979ad41e7a71b19f9e2b7ff99542134",
+        "manatee": "git+ssh://git@github.com:joyent/manatee.git#acbcf534d671d9f75a7c3092520f650bbfb6b803",
         "carrier": "0.2.0"
     },
     "sdcDependencies": {
